spec/spec_helper.rb

```diff
@@ -21,6 +21,9 @@
 require 'codeclimate-test-reporter'
 CodeClimate::TestReporter.start
 
+require 'webmock/rspec'
+WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
@kaptron Interesting - What networks were interfering with the tests to need only localhost to connect?